### DISCUSSION TITLE
Foreign Key in One to Many Relation

### DISCUSSION THREAD

**Discussion by iherwig** [directus/directus#4853] [Archive]
Hi. Let's say i have two collections, `Company` and `Employee` and a 1:n relation between these two (a company has many employees). When i edit a company in directus, i see a table showing the related employees and two buttons `Create New` and `Add Existing`. When i press `Create New`, a drawer opens and i can create a new employee. But in this view i still need to select the company from a dropdown list to set the foreign key in the employee table correctly. My question is, if this can be automated so that the foreign key is automatically set to the company, from which i pressed `Create New`. Maybe i overlooked a setting though.

Thanks and best regards,
Ingo

**Comment by Nitwel** [comment]
You can just ignore that field as it will get autofilled though I agree that it is not really obvious that it does that.
There is already an issue to add an option to hide this field but for now just ignore it.

**Reply by iherwig** [reply]
It works! Thanks a lot for the fast reply!
